singular.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * The template for displaying all pages, single posts and attachments
+ *
+ * This is a new template file that WordPress introduced in
+ * version 4.2. Note that it uses conditional logic to display
```

Comment:
OK, so considering Travis didn't run correctly on this PR and this typo I'll resubmit. Should I update the minimum requirement in back-compat.php with my PR as well? Seems prudent.